frmSettings.Designer.cs

```diff
@@ -54,6 +56,9 @@ partial class frmSettings {
             this.pnlParentThreadDescriptionFormat = new System.Windows.Forms.Panel();
             this.grpNamingStructure = new System.Windows.Forms.GroupBox();
             this.grpGeneral = new System.Windows.Forms.GroupBox();
+            this.lblSupportedSitesNote = new System.Windows.Forms.Label();
+            this.txtThreadStatusBoxThreshold = new System.Windows.Forms.TextBox();
+            this.ThreadStatusBox = new System.Windows.Forms.ComboBox();
```

Comment:
```suggestion
            this.cboThreadStatus = new System.Windows.Forms.ComboBox();
```
Just to follow the existing convention.